client/web/page.tt

```diff
@@ -13,10 +13,11 @@ $(function() {
     // and application key. You're all set to start working with your Mobile Service!
     //var client = new WindowsAzure.MobileServiceClient(
     //    "ZUMOAPPURL",
+    //    "ZUMOGATEWAYURL",
```

Comment:
For MobileApp QS, we have been re-arrranging this if block, so .NET includes a commented out localhost block, but both use the same  client by default to talk to the cloud service.  Is that what we want to do here?